src/Sentry.Unity/UnitySentryOptions.cs

```diff
@@ -91,6 +92,27 @@ public void WriteTo(Utf8JsonWriter writer)
             writer.Flush();
         }
 
+        public SentryOptions ToSentryOptions()
```

Comment:
I'm not sure this is the best way for us to go.

Maybe it's time we inherit from `SentryOptions`? We don't need to serialize the whole thing. We serialize what we use from `SentryWindow` only

Reply:
We probably should after looking at other SDKs. This will unblock `AddIntegration` facility.